src/main.rs

```diff
@@ -0,0 +1,61 @@
+use k2::{
+    benchmark::Benchmark, experiment::ExperimentBuilder, lang_impl::GenericScriptingVm,
+    limit::Limit, util::find_executable,
+};
+
+use clap::{App, Arg};
+
+fn main() {
+    let python_bin = find_executable("python");
```

Comment:
A note on `find_executable`: it's fine for an example setup, but we should encourage users to use full paths for real experiments. Relying on `${PATH}` can lead to running the wrong VM, and in these experimental setups you sometimes have many different versions of the same VM.

Should we add a comment perhaps?

Reply:
This one doesn't worry me too much, I must admit.

Reply:
I see. I don't mind adding a comment though.

[I used `find_executable` because I wanted this example to be roughly equivalent to [example.krun](https://github.com/softdevteam/krun/blob/master/examples/example.krun)]